src/intl/es.json

```diff
@@ -0,0 +1,63 @@
+{
+  "title": "Resume Builder",
+  "build_resume": "Construir",
+  "home": "Home",
+  "about": "About",
```

Comment:
```suggestion
  "about": "Sobre",
```

Right?

Reply:
What's the context? I couldn't find it in the webpage

Reply:
Ahh you're right, this is not used anymore! It was a link to a about page